### PR TITLE
Add unlimited string behavior description

### PR DESCRIPTION
**Pull request by pouyaist**

### CODE REVIEW

File: content/refguide/saphana.md [hunk 1]
```diff
@@ -20,3 +20,15 @@ For example, you have two associated entities, *Person* and *Address*, and they
 SAP HANA is case sensitive when doing string comparisons and checks. This is important when using functions such as `contains()`, `starts-with()`, and `ends-with()`, or when using equality(`=`) or inequality(`!=`) in xpath constraints.
 
 For example, `contains('OneTwo', 'one')` will return `false`.
+
+### Unlimited String Behavior
+
+SAP HANA does not support unlimited string when using equality(`=`) or inequality(`!=`) in xpath constraints. However, it supports on functions including `contains()`, `starts-with()`, and `ends-with()`.
+
+### Sorting on Very Long Strings
```
Comment (grootjans) [inline content/refguide/saphana.md]: Perhaps we can make `sorting` and `select distinct` a sub-header of Unlimited string behavior. We can then clarify the Long string behavior in the main section.

File: content/refguide/oracle.md [hunk 2]
```diff
@@ -16,3 +16,7 @@ This problem is fixed in Oracle 11.2.0.4\. See [https://support.oracle.com/epmos
 Selecting DISTINCT attributes of type String and size > 2000 characters is not supported by Mendix due to a known Oracle limitation of selecting DISTINCT columns with a CLOB data type. When you run into this limitation, you may encounter an exception in the logs with a message like this:
 
 `Error Msg = ORA-06502: PL/SQL: numeric or value error: character string buffer too small`
+
+### Unlimited String Behavior
```
Comment (grootjans) [inline content/refguide/oracle.md]: Oracle also has this behavior for long-limited-strings. It starts at 2000 chars if I am right.

File: content/refguide/saphana.md (hunk 1 above)
Comment (pouyaist) [reply content/refguide/saphana.md]: changed it also added more

File: content/refguide/oracle.md (hunk 2 above)
Comment (pouyaist) [reply content/refguide/oracle.md]: true, i thought i added it but apparently not... now it is there

File: content/refguide/saphana.md [hunk 3]
```diff
@@ -20,3 +20,17 @@ For example, you have two associated entities, *Person* and *Address*, and they
 SAP HANA is case sensitive when doing string comparisons and checks. This is important when using functions such as `contains()`, `starts-with()`, and `ends-with()`, or when using equality(`=`) or inequality(`!=`) in xpath constraints.
 
 For example, `contains('OneTwo', 'one')` will return `false`.
+
+### Unlimite and Long-limited String Behavior
```
Comment (grootjans) [inline content/refguide/saphana.md]: Unlimite -> Unlimited

Perhaps:
Behavior of Strings with unlimited and large size

File: content/refguide/oracle.md [hunk 4]
```diff
@@ -11,8 +11,21 @@ There is a known bug in older versions of Oracle 11.2 where converting numeric a
...
 This problem is fixed in Oracle 11.2.0.4\. See [https://support.oracle.com/epmos/faces/DocumentDisplay?id=9949330.8](https://support.oracle.com/epmos/faces/DocumentDisplay?id=9949330.8) and [http://stackoverflow.com/questions/16735793/strange-behavior-on-oracle-cast-to-nvarchar2](http://stackoverflow.com/questions/16735793/strange-behavior-on-oracle-cast-to-nvarchar2).
 
-### Select DISTINCT Attribute of Type CLOB
+
+
+### Unlimite and Long-limited String Behavior
+
+#### Comparison functions
+
+Oracle does not support unlimited strings or strings with a specified size > 2000\ characters when using equality(`=`) or inequality(`!=`) in xpath constraints. However, it supports on functions including `contains()`, `starts-with()`, and `ends-with()`.
```
Comment (grootjans) [inline content/refguide/oracle.md]: is the `\` behind the 2000 on purpose?

File: content/refguide/oracle.md [hunk 5]
```diff
@@ -11,8 +11,21 @@ There is a known bug in older versions of Oracle 11.2 where converting numeric a
...
+
+#### Sorting, Grouping, and Aggregating
+
+It is not possible to sort, group or use aggregate function such as `count()` on unlimited strings or strings with a specified length greater than 2000\. This is because such long or unlimited strings are implemented with the data type NCLOB. Consider decreasing the length of the string attribute or removing it from data grids.
```
Comment (grootjans) [inline content/refguide/oracle.md]: function -> functions

**Pull request merged**